src/components/Dropdowns/YearDropdown/index.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { MenuItem, TextField } from '@material-ui/core';
+import { getIn } from 'formik';
+
+const yearDropdownChoices = (minYear, maxYear) => {
+  const yearChoices = [];
+
+  for (let year = 0; year <= maxYear - minYear; year += 1) {
+    yearChoices.push(year + minYear);
+  }
+
+  return yearChoices;
+};
+
+/*
+ * From https://github.com/stackworx/formik-material-ui/tree/master/packages/formik-material-ui/src (TextField) :p
+ * This essentially takes the props received at the top-level <Field /> as well
+ * as the default props for that <Field /> (field, form, meta) and transforms
+ * all that into a props that is suitable for MUI's <TextField />
+ */
+export function fieldToTextField({
+  disabled,
+  field,
+  form: { isSubmitting, touched, errors },
+  ...props
+}) {
+  const fieldError = getIn(errors, field.name);
```

Comment:
what's getIn?

Reply:
https://stackoverflow.com/questions/56089054/how-to-dynamically-access-nested-errors-touched-on-formik-field

This is probably the only thing I found that actually explains what getIn() does lol. I don't know why there is no documentation for it